src/core/control/settings/SettingsEnums.h

```diff
@@ -69,6 +69,13 @@ enum StylusCursorType {
     STYLUS_CURSOR_ARROW = 3,
 };
 
+enum IconTheme {
+    ICON_THEME_LIGHT = 0,
+    ICON_THEME_DARK = 1,
+    ICON_THEME_LUCIDE_LIGHT = 2,
+    ICON_THEME_LUCIDE_DARK = 3,
```

Comment:
I think it's better to have separate options for style and dark/light:
* Keep the "dark theme" option in the settings (this is good for backward compatibility too -- upgrading won't cause users to suddenly lose their dark theme setting).
* Add a "style" option which allows you to select which icon theme you want. As a bonus, this also simplifies icon folder naming/handling, as you just need to define a icon folder name (e.g., iconLucide) and then append the `-dark` or `-light` based on the dark theme setting.

Reply:
Ok, that makes sense in view of the current behavior shown by Xournal++.